admin/app/components/complementary-certifications/attach-badges/badges/list.hbs

```diff
@@ -16,29 +11,70 @@
+          </ComplementaryCertifications::AttachBadges::Badges::Header>
+          <ComplementaryCertifications::AttachBadges::Badges::Header @isRequired="true">
+            <:default>Label du certificat</:default>
+            <:tooltip>
+              Renseignez un label qui permet de distinguer chaque RT (exemples : Pix+ Droit Expert, CléA Numérique, Pix+
+              Edu 2nd degré Confirmé etc…)
+            </:tooltip>
+          </ComplementaryCertifications::AttachBadges::Badges::Header>
+          <ComplementaryCertifications::AttachBadges::Badges::Header @isRequired="true">
+            <:default>Macaron de l'attestation PDF</:default>
+            <:tooltip>
+              Renseignez l’URL de l’image au format .pdf (fournie par les devs) pour l’attestation de certification PDF
+              du candidat.
+            </:tooltip>
+          </ComplementaryCertifications::AttachBadges::Badges::Header>
+          <ComplementaryCertifications::AttachBadges::Badges::Header>
+            <:default>Message du certificat</:default>
+            <:tooltip>
+              Renseignez le message définitif à afficher sur le certificat Pix App pour les certifications comportant
+              plusieurs volets (exemple : Vous avez obtenu la certification Pix+ Edu niveau “Avancé”).
+            </:tooltip>
+          </ComplementaryCertifications::AttachBadges::Badges::Header>
+          <ComplementaryCertifications::AttachBadges::Badges::Header>
+            <:default>Message temporaire certificat</:default>
+            <:tooltip>
+              Renseignez le message temporaire à afficher sur le certificat Pix App en attendant la validation de tous
+              les volets de la certification (exemple : Vous avez obtenu le niveau “Avancé” dans le cadre du volet 1 de
+              la certification Pix+Édu. Votre niveau final sera déterminé à l’issue du volet 2).
+            </:tooltip>
+          </ComplementaryCertifications::AttachBadges::Badges::Header>
         </tr>
       </thead>
       <tbody>
         {{#each @options as |option|}}
-          <tr aria-label="Résultat thématique {{option.id}} {{option.label}}">
-            <td>{{option.id}}</td>
-            <td>{{option.label}}</td>
-            <td>
-              <PixInput
-                @id="{{option.id}}"
-                placeholder="Exemple de niveau de RT : 1"
-                type="number"
-                min="1"
-                max="{{@options.length}}"
-                {{on "input" this.onUpdateLevel}}
-              />
-            </td>
-          </tr>
+          <ComplementaryCertifications::AttachBadges::Badges::Row
+            @badgeId={{option.id}}
+            @badgeLabel={{option.label}}
+            @badgeMaxLevel={{option.length}}
+            @onFieldUpdate={{this.onBadgeUpdated}}
+          />
         {{/each}}
       </tbody>
     </table>
+
+    <p aria-details="Les colonnes marquées avec un astérisque sont des champs obligatoires">
```

Comment:
**Remarque :** L'usage d'un aria-details n'est pas recommandé dans ce cas. 

`The aria-details attribute can be used to provide additional information or complex descriptions to an object`
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-details

Je propose plutôt : 
```suggestion
    <p 
      <span class="screen-reader-only">
         Les colonnes marquées avec un astérisque sont des champs obligatoires". 
      </span>
      Champs obligatoires
   </p>
```
https://1024pix.atlassian.net/wiki/spaces/DEV/pages/3290529806/Accessibilit#Comment-construire-son-lien-avec-une-information-suppl%C3%A9mentaire-qui-sera-restitu%C3%A9-mais-qui-ne-sera-pas-affich%C3%A9-visuellement-?